README.rst

```diff
@@ -25,29 +25,19 @@ Currently the following checks are available:
 Running
 -------
 
-You can run the checks locally with
-`Taskotron <https://fedoraproject.org/wiki/Taskotron>`__. First,
-install it (you can
-follow the
-`Quickstart <https://qa.fedoraproject.org/docs/libtaskotron/latest/quickstart.html>`__).
-You'll also need the ``rpm``, ``dnf`` and ``libarchive-c`` Python 2 modules
-(``python2-rpm``, ``python2-dnf``, ``python2-libarchive-c``).
-Note that Taskotron unfortunately runs on Python 2, but the code in
-this repository is Python 3 compatible as well.
-
-Once everything is installed you can run the task on a Koji build
-using the
-``name-(epoch:)version-release`` (``nevr``) identifier.
+To run this task locally, execute the following command as root (don't do this
```

Comment:
This is bad design, it worked before and now it requires this extra step (getting a system where this runs). Once I setup a mock based workaround for this, I'll provide a way to run it not only in the integration tests, but also manually.

Reply:
Agreed. I proposed some ideas how to mitigate this at least for local execution in the PR description.

Reply:
Added mock section bellow.